node/src/service.rs

```diff
@@ -240,6 +242,18 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 
     config.network.enable_dht_random_walk = true;
 
+    let node_peer_id = network.local_peer_id().to_string();
+
+    // Reference: https://substrate.dev/docs/en/knowledgebase/advanced/ss58-address-format
+    let decoded_peer_id = decode(node_peer_id)
+        .with_alphabet(Alphabet::BITCOIN)
+        .into_vec()
+        .unwrap();
+
+    let peer_id_hex = hex::encode(decoded_peer_id);
+
+    info!("PeerID (hex): {:?}", peer_id_hex);
+
```

Comment:
I think this is not required anymore. Since we have added new commandline option to do the same.

Reply:
Yeah, I thought about it, but I was thinking, it could also be a nice to have for someone who might not want to use a CLI. 

But I'll remove it.